Example/Pods/Analytics/Pod/Classes/Integrations/SEGScreenPayload.h

```diff
@@ -0,0 +1,18 @@
+#import <Foundation/Foundation.h>
+#import "SEGPayload.h"
+
+
+@interface SEGScreenPayload : SEGPayload
+
+@property (nonatomic, readonly) NSString *name;
+
+@property (nonatomic, readonly) NSString *category;
```

Comment:
I see that as of now, these properties are not being accessed using dot-notation or with getters/setters. But, for future-proofing, it might be a good idea to specify `copy` attribute in the property declarations.